internal/presenters/components.go

```diff
@@ -0,0 +1,166 @@
+package presenters
+
+import (
+	"bytes"
+	"fmt"
+	"slices"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
+)
+
+func RenderFindings(findings []Finding, showIgnored bool, showOpen bool) string {
+	if len(findings) == 0 {
+		return ""
+	}
+
+	response := ""
+
+	if showOpen {
+		response += RenderTitle("Open Issues")
+
+		for _, finding := range findings {
+			if finding.Ignored {
+				continue
+			}
+			response += RenderFinding(finding)
+		}
+	}
+
+	if showOpen && showIgnored {
+		response += RenderDivider()
+	}
+
+	if showIgnored {
+		response += RenderTitle("Ignored Issues")
+
+		for _, finding := range findings {
+			if !finding.Ignored {
+				continue
+			}
+			response += RenderFinding(finding)
+		}
+
+		response += RenderTip("Ignores are currently managed in the Snyk Web UI.\nTo edit or remove the ignore please go to: https://app.snyk.io/") + "\n"
```

Comment:
Suggestion: Replace the hard-coded URL by the configuration value `WEB_APP_URL` or even better a direct link to edit ignores if it is available.